source/localizable/_code-of-conduct.ru.md

```diff
@@ -0,0 +1,68 @@
+Цель
+-------
+
+Основная цель конференций и юзер-групп, ратифицировавших данный Кодекс Поведения: по возможности привлечь как можно большее число участников и участниц с наиболее разнообразными знаниями и опытом. Поэтому мы стремимся создать приветливую, комфортную и доброжелательную атмосферу для всех — вне зависимости от гендерной принадлежности, сексуальной ориентации, потенциала, этнической принадлежности, социально-экономического статуса и религиозных конфессий (или отсутствием таковых).
+
+Кодекc Поведения отражает наши ожидания от участников и участниц сообщества, а также последствия для тех, кто поведёт себя неприемлемо.
+
+Приглашаем участников и участниц наших мероприятий помочь нам создать комфортную и позитивную среду для всех.
+
+
+Открытый код / открытая культура / открытые технологии
```

Comment:
The original heading had “citizenship” which is missing in the translation, I'm not sure how important it is to keep it. The meaning is a bit different now.

Reply:
agree, will fix